x-pack/plugins/fleet/common/services/get_min_max_version.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { uniq } from 'lodash';
+import semverGt from 'semver/functions/gt';
+import semverLt from 'semver/functions/gt';
```

Comment:
did you mean to import from `semver/functions/lt` ?

Reply:
yes, strange that it didn't fail the tests